lib/ansible/modules/net_tools/snmp_facts.py

```diff
@@ -284,7 +296,7 @@ def main():
 
     # Verify that we receive a community when using snmp v2
     if m_args['version'] == "v2" or m_args['version'] == "v2c":
-        if m_args['community'] is False:
+        if m_args['community'] is None:
```

Comment:
This looks like more than a docs change.

Reply:
Correct, it is a cleanup. The defaults for this string was the boolean value False, which isn't a string. So we fall back to the default None, which is the same as unset.

That's the only way to correctly fix the validate-modules issue.